app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -743,6 +743,21 @@ struct DeviceDelegateOpenXR::State {
     // The rest of events shouldn't require any action from us, since we don't
     // modify existing objects and they should still be valid.
   }
+
+  bool CanEnablePassthrough() {
+    if (!passthroughSupported || passthrough == XR_NULL_HANDLE)
+      return false;
+
+    if ((passthroughState & XR_PASSTHROUGH_STATE_CHANGED_NON_RECOVERABLE_ERROR_BIT_FB) ||
+            (passthroughState & XR_PASSTHROUGH_STATE_CHANGED_RECOVERABLE_ERROR_BIT_FB)) {
+      return false;
```

Comment:
This would need to be refactored too

Reply:
Why? Note that this method will be called per-frame, and the runtime will emit the passthrough events when there are changes. If we don't store the state, we will need to keep flags anyway.

Reply:
I was referring to other devices not supporting the FB extension